src/AssetManager.Web/Startup.cs

```diff
@@ -21,6 +23,21 @@ public Startup(IConfiguration configuration)
 
         public IConfiguration Configuration { get; }
 
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddDbContext<AssetManagerContext>(options =>
+                options.UseSqlServer(Configuration.GetConnectionString("DefaultConnection")));
+
+            services.AddIdentity<User, IdentityRole>()
+                .AddEntityFrameworkStores<AssetManagerContext>()
+                .AddDefaultTokenProviders();
+
+            // Add application services.
+            //services.AddTransient<IEmailSender, EmailSender>();
+
+            services.AddMvc();
+        }
+
         // This method gets called when Development Environment is used
         // Use this method to set Development services, like Development database
```

Comment:
This method get invoked when we run the app from visual studio (Development Environment) 

We are using the same configuration of `Testing Environment` . 

Also check that we are configuring **Database** inside those **Environment Specific** methods. Not **Generic** `ConfigureServices` method.